scripts/wad-diff

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
How did I not know about shellcheck? Thanks for that. I'll try it on my future shell scripts. As to `#!/bin/sh` vs. `#!/bin/bash` - I agree that it's helpful to make the script to work on as many systems as possible, and they all have `/bin/sh`.  However, pure Bourne shell can have unpleasant syntax. Also, `/bin/sh` varies more system to system than `/bin/bash`. On Fedora it's often a symlink to `/bin/bash`. I believe on Ubuntu it can be a stripped down shell depending on the install. Also, this particular script will change significantly and become a part of wad2image.